libs/openFrameworks/utils/ofFileUtils.cpp

```diff
@@ -1804,28 +1804,21 @@ string ofFilePath::removeTrailingSlash(const of::filesystem::path & _path){
 
 
 //------------------------------------------------------------------------------------------------------------
-// FIXME: - re-avail
-string ofFilePath::getFileName(const of::filesystem::path& _filePath, bool bRelativeToData){
-	auto filePath = _filePath;
-
-	if(bRelativeToData){
-		filePath = ofToDataPathFS(filePath);
-	}
-
-	// FIXME: this is probably over complicated
-	return of::filesystem::path(filePath).filename().string();
+// FIXME: - Deprecate in favor of fs::path.filename()
+string ofFilePath::getFileName(const fs::path & filePath, bool bRelativeToData){
```

Comment:
Hey @ofTheo I've finally took time to review.
As this function only returns the filename of a specified fs::path the file we don't need the second parameter. Maybe we can deprecate this function with the second parameter and redesign to have only one?
fs::path can get fileName without even the file on path exists, it doesn't need to touch any file
makes sense?

Reply:
oh ha - yes! I don't know how I missed that. 
only case could be if someone does something like:  

```
ofFilePath::getFileName("../../", true); 

ofFilePath::getFileName("../../", false); 
```

might be good to test what the current code does vs the new code. 

but otherwise yeah to deprecating the second arg version. 🙂

Reply:
I've tested now and both in master and in PR the result is the same. empty string for both situations. 
I think this is the same case we discussed before of the name of a directory with trailing slash (similar to php?)

Reply:
I think now it is good to go, will wait for tests running.